src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -32,7 +32,12 @@ private static IntPtr LoadNativeLibrary()
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
                 {
                     libraryName = "libgdiplus.dylib";
+#if netcoreapp
```

Comment:
Nit: It would be nice to use the same condition in all places. .csproj file uses netcoreapp20, but the sources are using netcoreapp. Should we use netcoreapp20 everywhere?

Reply:
Switched to this pattern everywhere:

```c#
#if netcoreapp2.0
#else // use managed NativeLibrary API from .NET Core 3 onwards
#endif
```